specification/applicationinsights/resource-manager/microsoft.insights/stable/2015-05-01/favorites_API.json

```diff
@@ -0,0 +1,403 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApplicationInsightsManagementClient",
+    "description": "Azure Application Insights client for favorites.",
+    "version": "2015-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/components/{resourceName}/favorites": {  
+      "get": {
+        "description": "Gets a list of favorites defined within an Application Insights component.",
+        "operationId": "Favorites_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/FavoriteTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/SourceTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/CanFetchFavoriteContentParameter"
+          },
+          {
+            "$ref": "#/parameters/TagsFilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list containing 0 or more favorite definitions associated to the Application Insights component.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ApplicationInsightsComponentFavorite"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "FavoritesList": {
+            "$ref": "./examples/FavoritesList.json"
+          }
+        },
+        "x-ms-pageable": {
```

Comment:
the declared response type is not of pageable shape (object with `value` property of type array like in [this random example](https://github.com/Azure/azure-rest-api-specs-pr/blob/0bef2b99ad6d52781e3c308bc6b420f6eef59ef7/specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/subscriptions.json#L181-L189) - what you have is a bare array), i.e. either the `schema` is wrong or this annotation 🙂

Reply:
I checked the code in our service, but it never returns a paged item - only returns the entire list of favorites without chunking them into pages.
Removed the x-ms-pageable annotation.

I imagine this is also part of how we may be out of compliance with the ARM schemas at the moment (we are aware of this and future updates to the service are slated to bring us back into compliance, hence the back and forth with the ARM team earlier).